client/my-sites/email/email-providers-comparison/price/style.scss

```diff
@@ -4,6 +4,14 @@
 	margin-bottom: 12px;
 }
 
+.badge--info-green + .google-workspace-price__discount-badge.badge--info-green {
```

Comment:
I believe we are only applying the outline treatment to the extra discount? 

<img width="1067" alt="Screen Shot 2022-09-01 at 8 11 51 AM" src="https://user-images.githubusercontent.com/104910361/187911222-ca0f80e1-6dfb-4c80-86a1-3585dfe16878.png">

Reply:
@olaseni My bad, I didn't pull the latest while testing. Confirming that the badges are rendering as expected!

* One discount
<img width="887" alt="Screen Shot 2022-09-01 at 9 12 38 AM" src="https://user-images.githubusercontent.com/104910361/187923004-3aff9acb-0c00-45eb-9ee2-24f21463ac54.png">

* Extra discount
<img width="1044" alt="Screen Shot 2022-09-01 at 9 11 24 AM" src="https://user-images.githubusercontent.com/104910361/187923020-06bfacaa-b4d9-4dec-bf42-40b14e64661f.png">